delivery/src/main/java/com/storyous/delivery/common/db/model.kt

```diff
@@ -39,5 +39,15 @@ data class DeliveryOrderWithCustomer(
     @Embedded
     val order: DeliveryOrder,
     @Relation(parentColumn = "customerId", entityColumn = "id")
-    val customer: Customer
+    val customer: Customer,
+    @Relation(parentColumn = "deskId", entityColumn = "id")
+    val desk: Desk?
+)
+
+@Entity
+open class Desk(
+    val deskId: String,
```

Comment:
"deskId" can be primary key, so in DB will not be duplicates, bud next we will have to handle deleting desk. Do you have idea how to do it?